index.js

```diff
@@ -102,7 +105,7 @@ const renderPopUp = (id) => {
 function togglePopUp() {
   for (let i = 0; i < seeProject.length; i += 1) {
     seeProject[i].addEventListener('click', (e) => {
-      e.preventDefault();
+      // e.preventDefault();
```

Comment:
Great job team 🤝

- [ ] [Optional] Kindly remove commented code from your project even in the style sheet. Leaving commented code in your project is regarded as a bad practice.